client/src/components/App/Content/InvestigationForm/TabManagement/ExposuresAndFlights/FlightsForm/AirportInput/AirportInputStyles.ts

```diff
@@ -4,23 +4,11 @@ const useStyles = makeStyles({
     airportDetails: {
         display: 'flex',
         flexDirection: 'row',
-        justifyContent: 'space-between',
         marginLeft: '-5.5vw',
         '@media screen and (max-width: 1280px)': {
             marginLeft: '-20.5vw'
         },
     },
-    airportCountryTextField: {
-        width: '10vw',
-        '@media screen and (min-width: 1450px)': {
-            marginRight: '-6vw',
-        },
-    },
-    additionalAirportDetails: {
-        '@media screen and (min-width: 1800px)': {
```

Comment:
same here, i didn't put the divs you deleted for nothing

Reply:
it wasnt used